src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -328,6 +342,19 @@ const formConfig = {
             } explanation of benefits`,
           ...applicantInsuranceEOBSchema(false),
         },
+        secondaryScheduleOfBenefits: {
+          path: 'secondary-insurance-sob',
+          depends: formData =>
+            get('applicantHasSecondary', formData) &&
+            get('applicantSecondaryEOB', formData) === 'noEob',
+          title: formData =>
+            `${nameWording(formData)} ${
+              formData.applicantSecondaryProvider
+            } schedule of benefits`,
+          CustomPage: FileFieldWrapped,
+          CustomPageReview: null,
+          ...applicantInsuranceSOBSchema(false),
+        },
```

Comment:
Adds upload to secondary insurance screen and configures schema to use secondary language.